website/docs/docs/collaborate/data-health-signals.md

```diff
@@ -0,0 +1,87 @@
+
+## Data health signal criteria
+
+Each resource has a health state that is determined by specific set of criteria. Select the following tabs to view the criteria for that resource type.
+<Tabs>
+<TabItem value="models" label="Models">
+
+The health state of a model is determined by the following criteria:
+
+| **Health state** | **Criteria**   |
+|-------------------|---------------|
+| ✅ **Healthy**    | - Built successfully in the last run AND<br />- Built in the last 30 days AND<br />- Model has tests configured AND<br />- All tests passed AND<br />- All upstream sources are fresh or freshness is not applicable (set to `null`) AND<br />- Has a description |
+| 🟡 **Caution**   | - Not built in the last 30 days OR<br />- Tests are not configured OR<br />- Tests return warnings OR<br />- One or more upstream sources are stale:<br />&nbsp;&nbsp;&nbsp;&nbsp;- Has a freshness check configured<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check ran in the past 30 days<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check returned a warning<br />- Missing a description |
+| 🔴 **Degraded**  | - Model failed to build OR<br />- Model has failing tests OR<br />- One or more upstream sources are stale:<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check hasn’t run in the past 30 days<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check returned an error |
+| ⚪ **Unknown**    | - Unable to determine health of resource; no job runs have processed the resource.         |
+
+</TabItem>
+
+<TabItem value="sources" label="Sources">
+
+The health state of a source is determined by the following criteria:
+
+| **Health state** | **Criteria**   |
+|-------------------|---------------|
+| ✅ Healthy	| - Freshness check configured AND<br />- Freshness check passed AND<br />- Freshness check ran in the past 30 days AND<br />- Has a description |
+| 🟡 Caution	| - Freshness check returned a warning OR<br />- Freshness check not configured OR<br />- Freshness check not run in the past 30 days OR<br />- Missing a description |
+| 🔴 Degraded	| - Freshness check returned an error |
+| ⚪ Unknown	| Unable to determine health of resource; no job runs have processed the resource.     |
+
+</TabItem>
+
+<TabItem value="exposures" label="Exposures">
+
+The health state of an exposure is determined by the following criteria:
+
+| **Health state** | **Criteria**   |
+|-------------------|---------------|
+| ✅ Healthy	| - Underlying sources are fresh AND<br />- Underlying models built successfully AND<br />- Underlying models’ tests passing AND<br />- (TBD) Underlying models built in the last 30 days |
+| 🟡 Caution	| - At least one underlying source’s freshness checks returned a warning OR<br />- At least one underlying model was skipped OR<br />- At least one underlying model’s tests returned a warning OR<br />- (TBD) At least one model not built in the last 30 days |
```

Comment:
Could you remove the TBD points? These can be added in if/when we implement META-4025